spec/integration/api/transform_spec.rb

```diff
@@ -0,0 +1,247 @@
+require 'spec_helper'
+require 'securerandom'
+
+module Vault
```

Comment:
How are these tests ran? I ask since this feature is part of the ADP module, so I'm not sure how CI can spin up an enterprise version of Vault.

Reply:
The way the server is started up can be found here: https://github.com/hashicorp/vault-ruby/blob/master/spec/support/vault_server.rb

Basically, whatever version of vault is available on the machine using the `vault` command is the one that gets used. If the version of vault on the system is not enterprise version 1.4 or greater, it skips the test.

Reply:
I see. Can you paste the results of a local integration test run, perhaps in a collapsible section in the original PR post?

This could be done [like so](https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab):
<details>
  <summary>Click to expand!</summary>
  
  ```javascript
    function whatIsLove() {
      console.log('Baby Don't hurt me. Don't hurt me');
      return 'No more';
    }
  ```
</details>